picard/acoustid/json_helpers.py

```diff
@@ -34,16 +34,48 @@ def _make_releases_node(recording):
                 release_mb['title'] = release['title']
             else:
                 release_mb['title'] = release_group['title']
+
             if 'country' in release:
                 release_mb['country'] = release['country']
+
+            if 'date' in release:
+                release_mb['date'] = release['date']
+
+            if 'medium_count' in release:
+                release_mb['medium_count'] = release['medium_count']
+
+            if 'track_count' in release:
+                release_mb['track-count'] = release['track_count']
+
             release_mb['media'] = []
             for medium in release['mediums']:
                 medium_mb = {}
                 if 'format' in medium:
                     medium_mb['format'] = medium['format']
-                medium_mb['track-count'] = medium['track_count']
+
+                if 'track_count' in medium:
+                    medium_mb['track-count'] = medium['track_count']
+
+                if 'position' in medium:
+                    medium_mb['position'] = medium['position']
+
+                if 'tracks' in medium:
+                    medium_mb['tracks'] = medium['tracks'][0]
+
                 release_mb['media'].append(medium_mb)
-            release_list.append(release_mb)
+
+            # AcoustId service is returning country/date as attrib of the release, but really, according to MusicBrainz schema definition,
+            # its a one-many sub attribute in release events.
+            # They do return the releaseevents, but seem to be copying the first one on to the release.
+            # So if we have releaseevents, then use them to create multiple records, and ignore what is coming on the release
+            if 'releaseevents' in release:
+                for releaseevent in release['releaseevents']:
+                    release_mb['country'] = releaseevent.get('country', '')
```

Comment:
There is actually not country attribute in the MB response. Instead there is an area attribute looking something like this:

```
"area": {
    "iso-3166-1-codes": [
        "DE"
    ],
    "id": "85752fda-13c4-31a3-bee5-0e5cb1f51dad",
    "sort-name": "Germany",
    "disambiguation": "",
    "name": "Germany"
}
```

Overall I actually don't think we have much value of extracting this attribute here.

Reply:
I do see country in the mb response. It looks like this

"date": "1996-06-18",
**"country": "US",**
"release-events": [
	{
		"date": "1996-06-18",
		"area": {
			"id": "489ce91b-6658-3307-9877-795b68554c98",
			"name": "United States",
			"sort-name": "United States",
			"iso-3166-1-codes": [
				"US"
			]
		}
	}
],

Reply:
You are right. Must have been asleep, actually I just updated code surrounding this in https://github.com/metabrainz/picard/commit/800f61e6b6e239ac4cada0740ae68e04296a4154